.buildkite/pipeline.nightly.yml

```diff
@@ -34,3 +34,10 @@ steps:
     env:
       LOGGING_LEVEL: 4
       KUBERNETES_VERSION: v1.10-latest
+  - name: ':rubocop: RuboCop'
+    command: 'bin/ci bin/lint'
+    agents:
+      queue: k8s-ci
```

Comment:
Instead of using the specialized CI queue and running it in a container built by our own script, could we use a normal Shopify build queue + their default ruby container? I imagine that would be faster, and could actually run in parallel with the other steps (given our limited number of k8s-ci agents).

Reply:
I couldn't figure out how to use the shopify-build stuff in this file. I also wasn't sure we could have this style of pipline and a shopify-build one. Maybe I just couldn't find the right docs...